client/python/gradio_client/client.py

```diff
@@ -731,7 +731,7 @@ def _login(self, auth: tuple[str, str]):
             data={"username": auth[0], "password": auth[1]},
         )
         if not resp.is_success:
-            raise ValueError(f"Could not login to {self.src}")
+            raise AuthenticationError(f"Could not login to {self.src}")
```

Comment:
just fyi this will raise even when the problem isn't authentication, but the link being down for example because the space is sleeping. don't think it matters but if you want to make it strictly for authentication like you described in exceptions.py you should look at the status code.